src/libs/actions/Search.ts

```diff
@@ -61,4 +67,19 @@ function createTransactionThread(hash: number, transactionID: string, reportID:
     Onyx.merge(`${ONYXKEYS.COLLECTION.SNAPSHOT}${hash}`, onyxUpdate);
 }
 
-export {search, createTransactionThread};
+function holdMoneyRequestOnSearch(hash: number, transactionIDList: string[], comment: string) {
```

Comment:
I have created similar actions and params for api call here: https://github.com/Expensify/App/pull/41725/files

If you think the names make sense then you can rename yours. Or I will rename mine later

Reply:
> why are we adding this? the base ListItem already has a keyForList property.

Yes, but in `ListItem` it's optional, here it's mandatory, thanks to that I don't need to check it many times in components logic